routes/api.php

```diff
@@ -43,6 +44,11 @@
     Route::get('static/privacy', [PrivacyPolicyController::class, 'getPrivacyPolicy'])
          ->name('api.v1.getPrivacyPolicy');
 
+    Route::get("/health/ready", [HealthMonitorController::class, 'healthReady'])
+         ->name("api.health.ready");
+    Route::get("/health/fitness", [HealthMonitorController::class, 'healthFitness'])
+         ->name("api.health.fitness");
```

Comment:
Are you sure you want to make this information available to the public?

Reply:
I think it can be useful for our app developers, but I'm open for a discussion.

Reply:
non blocking for me :)
I don't know any case where this can be abused, so...